### ISSUE TITLE
False positives on integer division

### ISSUE THREAD

**Issue by hvelarde** [gforcada/flake8-pep3101#16]
the following code is causing the linter to fail:

```python
r = i % 4 * 64
g = i % 8 * 32
b = i % 16 * 16
```

see:

* https://github.com/collective/collective.cover/blob/master/src/collective/cover/testing.py#L107-L109
* https://travis-ci.org/collective/collective.cover/jobs/232977017

**Comment by gforcada** [comment]
It should check for the right hand operator to *not* be a number... patches welcome 👍 I don't have much time lately. 😞

**Comment by hvelarde** [comment]
is not that easy: the right hand operator could be also a variable; problem started with your latest release and this commit is my main suspect: https://github.com/gforcada/flake8-pep3101/commit/7e15700a70b3a75dfe31a13dbdb1dbbe8004f145

**Comment by sobolevn** [comment]
This function is also false positive:
```
def all_even_numbers():
    return (i for i in range(100) if i % 2 == 0)
```

**Comment by gforcada** [comment]
These should be fixed on #22 and released in 1.2, if you all could give it a try that would be awesome!

Thanks for using this flake8 plugin and taking the time to report! 🙇

**Comment by hvelarde** [comment]
@gforcada it's working, thanks!
